examples/object-detection/README.md

```diff
@@ -0,0 +1,134 @@
+# Using DIGITS to train an Object Detection network
+
+Table of Contents
+=================
+* [Introduction](#introduction)
+* [Dataset creation](#dataset-creation)
+    * [Preparing the data](#preparing-the-data)
+    * [Loading the data into DIGITS](#loading-the-data-into-digits)
+* [Model creation](#model-creation)
+    * [DetectNet](#detectnet)
+    * [Training DetectNet in DIGITS](#training-detectnet-in-digits)
+* [Verification](#verification)
+
+## Introduction
+
+In this tutorial we will see how DIGITS may be used to train an Object Detection neural network using the Caffe back-end.
+In this particular example, we will train the network to detect cars in pictures taken from a dashboard camera.
+During inference, object detection will be materialized by drawing bounding rectangles around the detected objects.
+
+## Dataset creation
+
+### Preparing the data
+
+This walk-through was tested using images and labels from the **Car** Object Detection track of the KITTI Vision benchmark.
+Other similar datasets may be used though you may prefer to download the KITTI dataset from their web site if you wish to replicate the results from this example.
+
+Optionally you may split the dataset into a training set and a (usually much smaller) validation set.
+Doing so is strongly recommended to assess the quality of the neural network.
+
+The data need to be structured in the following way:
+- An image folder contains supported images (.png, .jpg, .jpeg, .bmp, .ppm).
+- A label folder contains .txt files in KITTI format that define the ground truth.
+Note that for each image in the image folder there must be a corresponding text file in the label folder.
+For example if the image folder includes an image named `foo.png` then the label folder needs to include a file named `foo.txt`.
+
+There needs to be one of the above set of directories for each of the training and validation sets.
+
+### Loading the data into DIGITS
+
+To create an Object Detection dataset in DIGITS, [this version of `3.4-dev`](https://github.com/NVIDIA/DIGITS/tree/2613082eec6c207548c99bb69fa5defc9473b1e8), or any more recent version, is required.
+
+On the DIGITS home page, select the `Datasets` tab then click `Image>Object Detection`:
```

Comment:
FWIW, the dataset instructions are for advanced users only.  The naive user (like me) will probably have a hard time picking the right KITTI files/organizing them.